docs/provision.md

```diff
@@ -50,4 +50,4 @@ reboot:
 
      DONE! Click submit :tada:
 
-7. If you want to view the console while it boots, make sure to trust the CA certificate in your browser, it's hosted at https://centos7-katello-nightly.example.com/pub/katello-server-ca.crt, and you'll need to make sure you're accessing the katello via it's proper hostname (add an entry to /etc/hosts)
+7. If you want to view the console while it boots, make sure to trust the CA certificate in your browser, it's hosted at https://centos9-katello-nightly.example.com/pub/katello-server-ca.crt, and you'll need to make sure you're accessing the katello via it's proper hostname (add an entry to /etc/hosts)
```

Comment:
I think the actual hostname depends on what was chosen in step 1.

Reply:
To be explicit: currrently we only have `centos9-stream-katello-nightly` so at least I'd expect
```suggestion
7. If you want to view the console while it boots, make sure to trust the CA certificate in your browser, it's hosted at https://centos9-stream-katello-nightly.example.com/pub/katello-server-ca.crt, and you'll need to make sure you're accessing the katello via it's proper hostname (add an entry to /etc/hosts)
```